src/index.html

```diff
@@ -4,9 +4,439 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__inner-container">
+        <div class="header__logo">
+          <a 
+            href="./index.html"  
+            target="_blank"
+          >
+            <img src="./images/logo.png" alt="logo">
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__link">
+            <li class="nav__item">
+              <a 
+                href="#" 
+                class="nav__refer is-active" 
+                target="_blank"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a 
+                href="#" 
+                class="nav__refer" 
+                target="_blank"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
```

Comment:
```suggestion
        </div>
        
        <nav class="nav">
          <ul class="nav__link">
            <li class="nav__item">
              <a 
                href="#" 
                class="nav__refer is-active" 
                target="_blank"
              >
                Apple
              </a>
            </li>
            
            <li class="nav__item">
              <a 
                href="#" 
                class="nav__refer" 
                target="_blank"
              >
                Samsung
              </a>
            </li>
            
            <li class="nav__item">
```
Fix code style all over the page according to the checklist p.2

<img width="1048" alt="Screenshot 2022-12-07 at 15 33 59" src="https://user-images.githubusercontent.com/104769168/206192864-ca3c2e45-a8f3-46f4-8b74-93e8f2a9df58.png">